### PR TITLE
Rest

### PR DESCRIPTION
**Pull request by theScrabi**
I'm done, 

this is the whole database maped into the endpoionts descriped [here](https://fsin-ohm.github.io/Matomat-Documentation/rest_api). Login does not work yet, or is ignored, creating/changing/queering data is now possible.

You can use the collections in the postman folder to test the server.

### CODE REVIEW

File: mockup/mockup_server.py [hunk 1]
```diff
@@ -53,11 +53,11 @@ def admins0():
...
 
-@app.route("/v1/products", methods=["GET", "POST"])
-def products():
-    return returnFileOnGet("products.json")
+@app.route("/v1/prices", methods=["GET", "POST"])
+def prices():
+    return returnFileOnGet("prices.json")
 
-@app.route("/v1/products/0", methods=["GET", "PATCH", "DELETE"])
+@app.route("/v1/prices/0", methods=["GET", "PATCH", "DELETE"])
```
Comment (Nefrin) [inline mockup/mockup_server.py]: Prices should never be `DELETE`d

File: postman/admins.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+					"type": "basic",
+					"basic": [
+						{
+							"key": "password",
+							"value": "047595d0fae972fbed0c51b4a41c7a349e0c47bb",
+							"type": "string"
+						},
+						{
+							"key": "username",
+							"value": "",
```
Comment (Nefrin) [inline postman/admins.postman_collection.json]: empty username?

File: src/main/java/org/fsin/matomat/rest/model/Order.java [hunk 3]
```diff
@@ -1,22 +1,30 @@
 package org.fsin.matomat.rest.model;
 
 public class Order extends Transaction {
-    private Product[] products;
-    private long buy_cost;
+    private int admin;
+    private OrderedProduct purchased[];
```
Comment (Nefrin) [inline src/main/java/org/fsin/matomat/rest/model/Order.java]: This looks like C Syntax. Array Braces should be on the type. Does this compile?

Comment (Nefrin) [reply src/main/java/org/fsin/matomat/rest/model/Order.java]: > You can also place the brackets after the array's name:
> // this form is discouraged
> float anArrayOfFloats[];
> However, convention discourages this form; the brackets identify the array type and should appear with the type designation.

from [Java Tutorials](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html)

File: mockup/mockup_server.py (hunk 1 above)
Comment (theScrabi) [reply mockup/mockup_server.py]: All right.

**Pull request merged**